qa/workunits/rbd/rbd_consistency_groups_helpers.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/sh
```

Comment:
Nit: combine this w/ rbd_groups.sh -- a separate file isn't much use when you don't have multiple tests referencing common functions